launcher/src/main/java/com/skcraft/launcher/util/HttpRequest.java

```diff
@@ -109,31 +110,54 @@ public HttpRequest execute() throws IOException {
                 throw new IllegalArgumentException("Connection already executed");
             }
 
-            conn = (HttpURLConnection) reformat(url).openConnection();
-            conn.setRequestProperty("User-Agent", "Mozilla/5.0 (Java) SKMCLauncher");
+            String location = null;
+            URL connectUrl = url;
 
-            if (body != null) {
-                conn.setRequestProperty("Content-Type", contentType);
-                conn.setRequestProperty("Content-Length", Integer.toString(body.length));
-                conn.setDoInput(true);
-            }
+            for(int redirect = 0; redirect <= maxRedirectFollow; redirect++) {
+                conn        = (HttpURLConnection) reformat(connectUrl).openConnection();
 
-            for (Map.Entry<String, String> entry : headers.entrySet()) {
-                conn.setRequestProperty(entry.getKey(), entry.getValue());
-            }
+                conn.setConnectTimeout(15000);
+                conn.setReadTimeout(15000);
+                conn.setInstanceFollowRedirects(false);
+                conn.setRequestProperty("User-Agent", "SKCraft Launcher");
```

Comment:
Can this be reverted to the previous user-agent, `"Mozilla/5.0 (Java) SKMCLauncher"`